Source/Core/DiscIO/WiiWad.h

```diff
@@ -20,6 +20,7 @@ class WiiWAD
 public:
   explicit WiiWAD(const std::string& name);
   explicit WiiWAD(std::unique_ptr<BlobReader> blob_reader);
+  WiiWAD(WiiWAD&&) = default;
```

Comment:
Should the move assignment operator be defaulted as well for symmetry?

```cpp
WiiWAD& operator=(WiiWAD&&) = default;
```

Reply:
Good idea, I've added this and `operator!=` for `Content`.